test/README.md

```diff
@@ -50,8 +50,10 @@ end-to-end, or instrumented, tests. Unit tests do not require any particular
 environment to run in, but the instrumented tests, however, require a connected
 Android device or emulator to run in.
 
-To run the unit tests, run: `gradlew test`.
-To run the instrumented tests, run: `gradlew connectedAndroidTest`.
+- To run the unit tests, run: `gradlew test`.
```

Comment:
```suggestion
- Shell command to run the unit tests: `gradlew test`.
```

Reply:
"Shell" is not a very common word in Windows land.
Besides, we never explicitly mention that those are command line commands or similar, do we?